client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -73,7 +76,14 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			statusText = this.props.translate( 'scheduled' );
+			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
+			statusText = this.props.translate( 'scheduled %(scheduledTime)s', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?